client/js/components/procedure/admin/InstitutionTagManagement/OrganisationTable.vue

```diff
@@ -0,0 +1,711 @@
+          <dl class="pl-4 w-full">
+            <dt class="color--grey">
+              {{ Translator.trans('address') }}
+            </dt>
+            <template v-if="locationContacts && hasAdress(locationContacts)">
+              <dd
+                v-if="locationContacts.street"
+                class="ml-0">
+                {{ locationContacts.street }}
+              </dd>
+              <dd
+                v-if="locationContacts.postalcode"
+                class="ml-0">
+                {{ locationContacts.postalcode }}
+              </dd>
+              <dd
+                v-if="locationContacts.city"
+                class="ml-0">
+                {{ locationContacts.city }}
+              </dd>
+            </template>
+            <dd
+              v-else
+              class="ml-0">
+              {{ Translator.trans('notspecified') }}
+            </dd>
+          </dl>
+          <dl class="pl-4 w-full">
+            <dt class="color--grey">
+              {{ Translator.trans('phone') }}
+            </dt>
+            <dd
+              v-if="locationContacts?.hasOwnProperty('phone') && locationContacts.phone"
+              class="ml-0">
+              {{ locationContacts.phone }}
+            </dd>
+            <dd
+              v-else
+              class="ml-0">
+              {{ Translator.trans('notspecified') }}
+            </dd>
+            <dt class="color--grey mt-2">
+              {{ Translator.trans('email.participation') }}
+            </dt>
+            <dd
+              v-if="participationFeedbackEmailAddress"
+              class="ml-0">
+              {{ participationFeedbackEmailAddress }}
+            </dd>
+            <dd
+              v-else
+              class="ml-0">
+              {{ Translator.trans('no.participation.email') }}
+            </dd>
+            <template v-if="ccEmailAddresses">
+              <dt class="color--grey mt-2">
+                {{ Translator.trans('email.cc.participation') }}:
+              </dt>
+              <dd class="ml-0">
+                {{ ccEmailAddresses }}
+              </dd>
+            </template>
+            <template v-if="contactPerson">
+              <dt class="color--grey mt-2">
+                {{ Translator.trans('contact.person') }}:
+              </dt>
+              <dd class="ml-0">
+                {{ contactPerson }}
+              </dd>
+            </template>
+          </dl>
+          <dl
+            v-if="hasPermission('feature_institution_tag_read') && Array.isArray(assignedTags) && assignedTags.length > 0"
+            class="pl-4 w-full">
+            <dt class="color--grey">
+              {{ Translator.trans('tags') }}
+            </dt>
+            <dd class="ml-0">
+              <div class="flex flex-wrap gap-1 mt-1">
+                <span
+                  v-for="tag in assignedTags"
+                  :key="tag.id">
+                  {{ tag.name }}
+                </span>
+              </div>
```

Comment:
I guess you just moved this, but I find the way tags are displayed a little confusing - they should at least be comma-separated